### ISSUE TITLE
How to penalize the loss of one class more than the other in tensorflow for a multi class problem?

### ISSUE THREAD

**Issue by ram1897** [tensorflow/tensorflow#22343]
Let's say my model has two classes Class 1 and Class 2. Both Class 1 and Class 2 has a equal amount of training and testing data. But I want to penalize the loss of the Class 1 more than Class 2, so that one class has a fewer number of False Positives than the other (I want the model to perform better for one class than the other).

**Comment by harshini-gadige** [comment]
This question is better asked on StackOverflow since it is not a bug or feature request. There is also a larger community that reads questions there.
